src/daemon/variables_entrypoint.sh

```diff
@@ -102,6 +102,12 @@ if [[ "$KV_TYPE" == "etcd" ]]; then
   ETCDCTL_OPTS=(--peers ${ETCD_SCHEMA}${KV_IP}:${KV_PORT})
 fi
 
+if is_available python; then
+  PYTHON=python
+else
+  PYTHON=python3
```

Comment:
Why are we assuming that if `python` is not present then `python3` is?
Shouldn't we do something like this perhaps?

```
PYTHON=python
if is_available python3; then
  PYTHON=python3
fi
```

Or check for the existence of each and then fail if not present?

Reply:
> Why are we assuming that if python is not present then python3 is?

because on distribution (like rhel 8, fedora 29) where python3 is the default python interpreter there's no python command (or symlink).

> Or check for the existence of each and then fail if not present?

We could but it's probably safe to say that base distro container images have at least one python interpreter

Reply:
Looks acceptable.